src/xcode/ENA/ENA/Source/Scenes/ExposureSubmission/ExposureSubmissionCoordinator.swift

```diff
@@ -566,17 +566,17 @@ class ExposureSubmissionCoordinator: NSObject, ExposureSubmissionCoordinating, R
 		alert.addAction(
 			UIAlertAction(
 				title: AppStrings.ExposureSubmissionTestResultAvailable.closeAlertButtonClose,
-				style: .cancel,
-				handler: { [weak self] _ in
-					self?.dismiss()
-				}
+				style: .cancel
 			)
 		)
 
 		alert.addAction(
 			UIAlertAction(
 				title: AppStrings.ExposureSubmissionTestResultAvailable.closeAlertButtonContinue,
-				style: .default
+				style: .default,
+				handler: { [weak self] _ in
+					self?.dismiss()
+				}
```

Comment:
This change does not make any sense to me. If you tab "Weiter", we cancel? And if you tap "Abbrechen", we continue? I don't think this is correct.

Reply:
@nickguendling This what is specified in the ticket. The popup says you are about to cancel. Do you want to cancel [Abbrechen] or continue [Weiter] with the cancellation.

Reply:
I have contacted the PO/UX/UA directly to clarify this, it's been a long discussion already 😅

Reply:
"Weiter" is a terrible and ambiguous term in such a situation. This should be changed and I hope the discussion will solve that.